VocaDbModel/Service/ReleaseEventService.cs

```diff
@@ -34,6 +34,25 @@ public ReleaseEventService(ISessionFactory sessionFactory, IUserPermissionContex
 
 		}
 
+		public VenueWithEventsContract[] GetReleaseEventsByVenues() {

```

Comment:
You should move this to EventQueries class instead, unless there's a specific reason adding it here. These legacy "services" are tied to NHibernate, and not testable.